docs/source/sections/HowToGetStartedonISB-CGC.rst

```diff
@@ -106,7 +106,7 @@ I. Getting Started:
       `About ISB-CGC Cloud-Hosted Datasets <https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/Hosted-Data.html>`_
       
  2.) To access controlled-access data, dbGaP authorization is required.
-      `Accessing Controlled-Access Data and acquiring dbGaP authorization <https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/Hosted-Data.html>`_
+      `Accessing Controlled-Access Data and acquiring dbGaP authorization <https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/webapp/Gaining-Access-To-Contolled-Access-Data.html#interactive-access-to-controlled-data>`_
```

Comment:
We should go with the new link. I don't have the ability to easily resolve complicated merges...